site/www/vendors/vanta/viz.js

```diff
@@ -346,7 +350,7 @@ class Viz {
     }
 
     this.renderer.setSize(this.el.offsetWidth, this.el.offsetHeight)
-    this.renderer.setPixelRatio(window.devicePixelRatio)
+    this.renderer.setPixelRatio(getPixelRatio())
```

Comment:
Isn't the central issue that the lines are too thick with pixel ratio is 1?

On a screen with pixel ratio of 1, the lines and, e.g., the font use the same definition of pixel: CSS Pixel, which is 1:1 with device pixel.

On a screen with a pixel ratio of 2, the lines and the font, I presume, use different definitions of pixel. The font still uses CSS pixel. However, thanks to `setPixelRatio`, I assume, the `material.linewidth` is now interpreted in terms of device pixels which are 2:1::device:css. Ergo, the lines look thinner compared to the font (or anything else measured in device pixels) on a higher DPI screen than on a lower DPI screen.

It seems to me that we ought to scale `material.linewidth` with the pixel ratio?

Reply:
Does not work

Reply:
Also line width is inconsistently applied, differs between platforms. Apparently windows fixes to 1 regardless of setting, not that this affects us, since we want < 1